templates/localvolumeprovisioner.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: kubeaddons.mesosphere.io/v1alpha1
 kind: Addon
 metadata:
   name: localvolumeprovisioner
-  namespace: kubeaddons
+  namespace: kube-system
```

Comment:
Thats the wrong namespace, see https://github.com/mesosphere/kubeaddons-configs/blob/master/templates/awsebscsiprovisioner.yaml#L15

Reply:
Yes, currently the addon itself will be in kubeaddons even if the actual deployment ends up in kube-system. Improvements for this are suggested in [https://jira.mesosphere.com/browse/DCOS-57136](DCOS-57136) which could potentially be the follow up to this for later iterations.

Reply:
good catch!